Tools/Pipeline/MainWindow.cs

```diff
@@ -66,6 +66,10 @@ public void Attach(IController controller)
 
             cmdDebugMode.Checked = PipelineSettings.Default.DebugMode;
             CmdDebugMode_Executed(this, EventArgs.Empty);
+            
+            // MultithreadMode is experimental and disabled on startup.
+            //cmdMultithreadMode.Checked = PipelineSettings.Default.MultithreadMode; 
```

Comment:
There really is no point in not allowing the setting for Multithreaded Mode to be saved between instances...